src/libs/actions/IOU.ts

```diff
@@ -5554,7 +5554,24 @@ function deleteTrackExpense(chatReportID: string, transactionID: string, reportA
         return deleteMoneyRequest(transactionID, reportAction, isSingleTransactionView);
     }
 
-    const {parameters, optimisticData, successData, failureData, shouldDeleteTransactionThread} = getDeleteTrackExpenseInformation(chatReportID, transactionID, reportAction);
+    const chatReportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReportID}`] ?? {};
+    const whisperAction = Object.values(chatReportActions).find(
```

Comment:
Please move the functionality of fetching the Track Expense Actionable Whisper to a helper function in ReportActionUtils.ts with something like `getTrackExpenseActionableWhisper` as that would be neater.